### PR TITLE
ENH: added collapse widget to feature importance viz results

### LINKED ISSUE #61: display important features lists as collapsible blocks of content
Is this even possible if the lists are provided as tables (from pandas DataFrames)?

### PR DESCRIPTION
**Pull request by nbokulich**
fixes #61

### CODE REVIEW

File: q2_sample_classifier/assets/index.html [hunk 1]
```diff
@@ -70,7 +70,22 @@
       <a href="feature_importance.tsv">
         <p>Download as tsv</p>
       </a>
-      {{ importances }}
+    </div>
+    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```
Comment (thermokarst) [inline q2_sample_classifier/assets/index.html]: Can you nest this panel `div` into the `<div class="col-lg-12">` above? That way the panel will fit inside of the grid container.

before:
screen shot 2017-09-06 at 11 37 03 am

after:
screen shot 2017-09-06 at 11 37 27 am

File: q2_sample_classifier/assets/index.html [hunk 2]
```diff
@@ -70,7 +70,22 @@
...
+      <div class="panel panel-default">
+        <div class="panel-heading" role="tab" id="accordion-heading">
+          <h4 class="panel-title">
+            <a role="button" data-toggle="collapse" data-parent="#accordion" href="#table" aria-expanded="true" aria-controls="table">
+              Collapse
```
Comment (thermokarst) [inline q2_sample_classifier/assets/index.html]: Can you rename to 'Click to Collapse/Expand'? Users might not be familiar with this type of functionality so explicitly pointing out the behavior might be useful.

**Comment by nbokulich** [comment]
thanks @thermokarst ! I have fixed those two issues as you requested...

**Comment by thermokarst** [comment]
@nbokulich, I took a note from @ebolyen's [recent review](https://github.com/qiime2/q2-longitudinal/pull/33#discussion_r137366631), and rather than put you through another round of back-and-forth I just made the changes directly in the PR. Thanks!

**Pull request merged**